op_builder/async_io.py

```diff
@@ -50,10 +50,9 @@ def extra_ldflags(self):
         return ['-laio']
 
     def is_compatible(self):
-        aio_libraries = ['libaio-dev']
-        aio_compatible = self.libraries_installed(aio_libraries)
+        aio_compatible = self.has_function('io_submit', ('aio', ))
```

Comment:
Can you please a brief explanation for `io_submit`?

Reply:
If that's too much or not quite right, just let me know.